stock_inventory_analytic/__manifest__.py

```diff
@@ -4,13 +4,15 @@
     'name': "Stock Inventory Analytic",
     'summary': """
         Stock Inventory Analytic """,
-    'author': 'Eficent, Odoo Community Association (OCA)',
+    'author': 'brain-tec, Eficent, Odoo Community Association (OCA)',
```

Comment:
why?

Reply:
Do you mean *why the authorship*? If so, it's because of the migration. Aren't migrations considered when authoring a module?

Reply:
@BT-cserra not usually, as a migration implies very little effort to be done. Only a considerable amount of work would count for being considered co-author of the module.

Reply:
@mreficent done. FYI: @pedrobaeza